__tests__/app.test.js

```diff
@@ -131,3 +131,33 @@ test('should have articles in decending order ', () => {
       });
 });
 });
+
+describe('should give back comments most recently created  ', () => {
+
+  test(' test to see we are recieving an array of comments', () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then((response) => { const {comments} = response.body;
+// console.log(response.body, "THIS IS response");
+    expect(Array.isArray(comments)).toBe(true)
```

Comment:
This is not actually checking we are receiving an array of comments. We need to assert that every comment has all of the key value pairs we expect. I would recommend looking into .toMatchObject to do this.